test/studies/uts/brg_sha1.c

```diff
@@ -111,14 +111,14 @@ int rng_nextrand(RNG_state *mystate){
 
 /* condense state into string to display during debugging */
 char * rng_showstate(RNG_state *state, char *s){
-  sprintf(s,"%.2X%.2X...", state[0],state[1]);
+  snprintf(s, 8 * sizeof(char), "%.2X%.2X...", state[0],state[1]);
```

Comment:
This is another place to make sure you have enough characters for the null terminator where is 8 coming from?)

Reply:
8 is coming from two two-digit hex numbers (4 chars), plus 3 literal `.`s (3 chars), plus null terminator (1 char). Did I miss something?

Reply:
Nope, you didn't, I was just making sure. Looks good!